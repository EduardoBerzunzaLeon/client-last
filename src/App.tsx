import { Provider } from 'react-redux';
import PrimeReact from 'primereact/api';

import { Router } from './router/Router';
import { store } from './redux/store';

import 'primeflex/primeflex.css';
import 'primeicons/primeicons.css';
import 'primereact/resources/primereact.css';
import 'primereact/resources/themes/bootstrap4-light-purple/theme.css';

import './app.scss';

function App() {
  PrimeReact.ripple = true;
  return (
    <Provider store={store}>
      <div>
        <Router />
      </div>
    </Provider>
  );
}

export default App;
