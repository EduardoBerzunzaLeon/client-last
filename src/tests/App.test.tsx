import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import App from '../App';

describe('<App /> Tests', () => {
  let wrapper: ShallowWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  test('Should render <App/> correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('Should render a div', () => {
    const div = wrapper.find('div');
    expect(div).toHaveLength(1);
  });
});
