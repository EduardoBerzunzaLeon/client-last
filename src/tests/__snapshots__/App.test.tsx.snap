// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Tests Should render <App/> correctly 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <div>
    <Router />
  </div>
</Provider>
`;
